src/Xamarin.Android.Build.Tasks/Tasks/Generator.cs

```diff
@@ -195,7 +195,7 @@ protected override string GenerateCommandLineCommands ()
 			}
 
 			cmd.AppendSwitch (ApiXmlInput);
```

Comment:
I think `ApiXmlInput` *also* needs quoting.  Not sure why it apparently didn't need it before, but *also* not sure where `Test Config` is coming from.

Ah; here: https://github.com/xamarin/xamarin-android/blob/e4debf72015549e6e19cbfb247a47fead6ce7a01/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs#L1124